tests/codeception/_support/E2eTester.php

```diff
@@ -44,4 +44,9 @@ public function dontSeeHorizontalScrollbars(){
             'Horizontal scrollbar'
         );
     }
+
+    public function pauseExecution()
```

Comment:
To ensure backward compatibility. **pauseExecution** was [removed in Version 3.0](https://github.com/Codeception/Codeception/blob/3.0/CHANGELOG-3.0.md) _`[...] pauseExecution removed in favor of $I->pause()`_